src/lang/pt-br.json

```diff
@@ -9,103 +9,104 @@
 	"ACTION.SHOVE": "Derrubar",
 	"ACTION.UNARMED_STRIKE": "Ataque desarmado",
 
-	"ACTOR.TypeCharacter": "Personagem",
-	"ACTOR.TypeMonster": "Monstro",
-	"ACTOR.TypeStronghold": "Fortaleza",
-
-	"ARMOR.BODY": "Corpo",
-	"ARMOR.HELMET": "Elmo",
-	"ARMOR.PART": "Parte do Corpo",
-	"ARMOR.RATING": "Valor de Armadura",
-	"ARMOR.SHIELD": "Escudo",
-	"ARMOR.TOTAL": "Total",
-
-	"ARTIFACT.APPEARANCE": "Aparência",
-	"ARTIFACT.ARTIFACT_BONUS": "Bônus de Artefato",
-	"ARTIFACT.DESCRIPTION": "Descrição",
-	"ARTIFACT.DRAWBACK": "Inconveniente",
-	"ARTIFACT.EPIC": "Épico",
-	"ARTIFACT.LEGENDARY": "Lendário",
-	"ARTIFACT.MIGHTY": "Poderoso",
-	"ARTIFACT.REGULAR": "Item normal",
-
-	"ATTACK.BLUNT": "Esmagamento",
-	"ATTACK.FEAR": "Medo",
-	"ATTACK.OTHER": "Outros",
-	"ATTACK.SLASH": "Cortante",
-	"ATTACK.STAB": "Golpear",
-
-	"ATTRIBUTE.AGILITY": "Agilidade",
-	"ATTRIBUTE.EMPATHY": "Empatia",
-	"ATTRIBUTE.STRENGTH": "Força",
-	"ATTRIBUTE.WITS": "Astúcia",
-
-	"BIO.AGE": "Idade",
-	"BIO.BODY": "Corpo",
-	"BIO.CLOTHING": "Roupas",
-	"BIO.DARK-SECRET": "Segredo Sombrio",
-	"BIO.EXPERIENCE": "Experiência",
-	"BIO.FACE": "Rosto",
-	"BIO.KIN": "Ascendência",
-	"BIO.NAME": "Nome",
-	"BIO.NOTE": "Notas",
-	"BIO.PRIDE": "Orgulho",
-	"BIO.PROFESSION": "Profissão",
-	"BIO.REPUTATION": "Reputação",
-	"BIO.WILLPOWER": "Força de Vontade",
-
-	"BUILDING.DESCRIPTION": "Descrição",
-	"BUILDING.EFFECT": "Efeito",
-	"BUILDING.NAME": "Nome",
-	"BUILDING.NUMBER": "Número",
-	"BUILDING.RAW_MATERIALS": "Estoque",
-	"BUILDING.REPUTATION": "Reputação",
-	"BUILDING.REQUIREMENT": "Requisitos",
-	"BUILDING.TIME": "Tempo",
-	"BUILDING.TOOLS": "Ferramentas",
-
-	"CARRYING_CAPACITY": "Capacidade de Carga",
-
-	"CONDITION.COLD": "Hipotérmico",
-	"CONDITION.HUNGRY": "Faminto",
-	"CONDITION.SLEEPY": "Insone",
-	"CONDITION.THIRSTY": "Desidratado",
-
-	"CONFIG.ALTERNATIVESKULLS": "Interação de ficha alternativa",
+  "ACTOR.TypeCharacter": "Personagem",
+  "ACTOR.TypeMonster": "Monstro",
+  "ACTOR.TypeParty": "Grupo",
+  "ACTOR.TypeStronghold": "Fortaleza",
+
+  "ARMOR.BODY": "Corpo",
+  "ARMOR.HELMET": "Elmo",
+  "ARMOR.PART": "Parte do Corpo",
+  "ARMOR.RATING": "Valor de Armadura",
+  "ARMOR.SHIELD": "Escudo",
+  "ARMOR.TOTAL": "Total",
+
+  "ARTIFACT.APPEARANCE": "Aparência",
+  "ARTIFACT.ARTIFACT_BONUS": "Bônus de Artefato",
+  "ARTIFACT.DESCRIPTION": "Descrição",
+  "ARTIFACT.DRAWBACK": "Efeitos Colaterais",    
+  "ARTIFACT.EPIC": "Épico",
+  "ARTIFACT.LEGENDARY": "Lendário",
+  "ARTIFACT.MIGHTY": "Poderoso",
+  "ARTIFACT.REGULAR": "Item normal",
+
+  "ATTACK.BLUNT": "Esmagamento",
+  "ATTACK.FEAR": "Medo",
+  "ATTACK.OTHER": "Outros",
+  "ATTACK.SLASH": "Cortante",
+  "ATTACK.STAB": "Golpear",
+
+  "ATTRIBUTE.AGILITY": "AGILIDADE",
+  "ATTRIBUTE.EMPATHY": "EMPATIA",
+  "ATTRIBUTE.STRENGTH": "FORÇA",
+  "ATTRIBUTE.WITS": "ASTÚCIA",
```

Comment:
The attributes should not be all caps. If they are the main tab on the character sheet will overflow and scroll as a result.